### PR TITLE
Add acronym exercise

### PR DESCRIPTION
**Pull request by zacchaeusluke**
I've ported the acronym exercise

### CODE REVIEW

**Comment by katrinleinweber** [comment]
Awesome, thank you so much :-) It runs fine for me, also while refactoring the example to a magrittr pipeline.

File: config.json [hunk 1]
```diff
@@ -358,6 +358,17 @@
         "control_flow_loops",
         "text_formatting"
       ]
+    },
+    {
+      "slug": "acronym",
+      "uuid": "54e60174-4833-418c-b9af-2fcb5722b1d6",
+      "core": false,
+      "unlocked_by": null,
```
Comment (katrinleinweber) [inline config.json]: Do you consider it the simplest `strings` or `filtering` exercise? If so, should we let a similar one be unlocked by this?

**Comment by katrinleinweber** [comment]
Awesome, thank you so much :-) It runs fine for me, also while refactoring the example to a magrittr pipeline.

File: exercises/acronym/acronym.R [hunk 2]
```diff
@@ -0,0 +1,3 @@
+acronym <- function(input) {
+    
+}
```
[the remark this answers is not in the document]
Comment (jonmcalder) [reply exercises/acronym/acronym.R]: I agree.

**Comment by jonmcalder** [comment]
Thanks for this PR @zacchaeusluke!

**Comment by katrinleinweber** [comment]
I'm fine with merging this now :-)

File: config.json (hunk 1 above)
Comment (katrinleinweber) [reply config.json]: Let's do this in another PR: We have a few more "floating" exercises that we should attach somewhere.

**Comment by katrinleinweber** [comment]
Thank you @zacchaeusluke :-)

**Pull request merged**